packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -243,6 +243,32 @@ describe('StripeHelper', () => {
     });
   });
 
+  describe('removeSources', () => {
+    it('removes all the sources', async () => {
+      const ids = {
+        data: [{ id: uuidv4() }, { id: uuidv4() }, { id: uuidv4() }],
+      };
+      sandbox.stub(stripeHelper.stripe.customers, 'listSources').resolves(ids);
+      sandbox.stub(stripeHelper.stripe.customers, 'deleteSource').resolves({});
+      const result = await stripeHelper.removeSources('cust_new');
+      assert.deepEqual(result, [{}, {}, {}]);
+      sinon.assert.calledThrice(stripeHelper.stripe.customers.deleteSource);
+    });
```

Comment:
nit-ish:
 - the test above doesn't check to see if the ids passed into `deleteSource` are the ones from the test.
 - the empty sources list branch of the code in `removeSources` isn't tested.